### PR TITLE
Added mandatory FirebaseOptions fo JavaDoc example.

### PR DESCRIPTION
**Pull request by andirayo**

### CODE REVIEW

**Comment by Feiyang1** [comment]
@egilmorez PTAL? related to b/150255991

**Comment by Feiyang1** [comment]
@andirayo probably want to update this line as well - https://github.com/firebase/firebase-js-sdk/pull/2778/files#diff-1709e2b1e06366271895c10a774faf4aL1132

File: packages/firebase/index.d.ts [hunk 1]
```diff
@@ -1117,6 +1117,8 @@ declare namespace firebase {
    * // https://console.firebase.google.com
    * firebase.initializeApp({
    *   apiKey: "AIza....",                             // Auth / General Use
+   *   applicationId: "1:27992087142:android:ce....",  // General Use
```
Comment (samtstern) [inline packages/firebase/index.d.ts]: Nit: change `:android:` to `:web:` ?

Comment (andirayo) [reply packages/firebase/index.d.ts]: Thanks, fixed.

**Pull request merged**